includes/specials/SpecialSearch.php

```diff
@@ -232,36 +240,6 @@ public function showResults( $term ) {
 
 		$out = $this->getOutput();
 
-		// Wikia change /Begin (ADi)
-		if ( ( $search instanceof SearchErrorReporting ) && $search->getError() ) {
-			$out->addWikiText( '==' . $search->getError() . '==' );
-			$out->addHTML( $search->getErrorTracker() );
-			$out->addHTML( $this->powerSearchBox( $term ) );
-			wfProfileOut( __METHOD__ );
-
-			return;
-		}
-		// Wikia change /End (ADi)
-
-		if ( $wgDisableTextSearch ) {
```

Comment:
`wgDisableTextSearch` is always false, I've checked in app, config and WikiFactory (report page).